db_holder.py

```diff
@@ -0,0 +1,41 @@
+from peewee import *
+
+db = SqliteDatabase('facts.db')
+
+
+class Fact(Model):
+    factName = CharField()
+
+    class Meta:
+        database = db  # This model uses the "people.db" database.
+
+
+arr = ['Россия — единственная в мире страна омываемая целой дюжиной морей.', 'На Алтае более восьмисот ледников.',
+       'Самое глубокое в мире метро находится в Санкт-Петербурге его глубина составляет более сотни метров.',
+       'Самый большой в мире активный вулкан — Ключевская сопка высотой в без малого пять километров. Он извергается уже более семи тысяч лет.',
+       'В метрополитене Москвы почти десять тысяч поездов.',
+       'Самая обширная равнина на планете — Западно-Сибирская.',
+       'Самый посещаемый Макдональдс в мире расположен в Москве на Пушкинской площади.',
+       'Суши в России популярнее чем в Японии.', 'До революции огнестрельное оружие в России продавалось свободно.',
+       'Останкинская телебашня — была самым высоким зданием в Европе долгое время. Но в последние годы появились небоскрёбы которые превзошли её по высоте.',
+       'На гербе Челябинска изображён верблюд.']
+
+
+def returnData():
+    db.connect()
+    db.drop_tables([Fact])
+    db.create_tables([Fact])
+    facts = []
+    for item in arr:
+        Fact.create(factName=item).save()
+
+    for person in Fact.select():
+        facts.append(person.factName)
+
+
+    db.close()
+    return facts
```

Comment:
every time you run this function, you add 9 new lines to table. with the time it becomes so heavy, that `Fact.select()` would take a minute.

it is better to fill db single time, than just to use it

Reply:
**(that is a critical bug)**